components/alert.css

```diff
@@ -0,0 +1,51 @@
+
+
+.alerttypes{
+    width: 80%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    
+    background-color:var(--offline-color);
+    padding: 2rem;
```

Comment:
Try using px for margin and padding coz general we don't want to enlarge them as screen enlarged